docs/guides/setup-channel.mdx

```diff
@@ -121,20 +121,63 @@ print(result)
 
-3. Optionally obtain a temporary access token by supplying the API token obtained above. This is especially useful if you would like control over tokens, such as token invalidation. Alternatively, you can work directly with your IBM Quantum Platform API token.
+    To set the IQP_API_TOKEN environment variable in your system, you can add the following line to your shell profile (for example, .bashrc or .zshrc) or by setting it directly in your terminal:
+
+    ```shell
+    export IQP_API_TOKEN=<your-token>
+    ```
+    When you invoke the environment variable in your code, include `import os`, as in this example request:
+
+    ```python
+    import os
+    import requests
+    
+    reqUrl = "https://api.quantum-computing.ibm.com/runtime/jobs?limit=10&offset=0&exclude_params=true"
+    
+    headersList = {
+      "Accept": "application/json",
+      "Authorization": f"Bearer {os.environ['IQP_API_TOKEN']}"
+    }
+    
+    payload = ""
+    
+    response = requests.request("GET", reqUrl, data=payload,  headers=headersList)
+    
+    print(response.json())
+    ```
+
+    Note that when creating an environment variable, your API token is still stored locally in plain text, and should be safeguarded.
+
+1. <span id="rest-untrusted"></span>**Avoid executing code on an untrusted machine or an external cloud Python environment to minimize security risks.** If you must use an untrusted environment (on, for example, a public computer), change your API token after each use by rotating it on the [IBM Quantum Platform dashboard](https://quantum.ibm.com/) (click the refresh button in the API token field). To initialize the service in this situation, use your API token directly: 
+
+    ```python
+    # Refresh your API token on the dashboard after using the code as follows:
+    "Authorization": "Bearer <your-token>" 
+    ```
+
+    <Admonition type="caution">
+      When sharing code with others, ensure that your API token is not embedded directly within the Python script. Instead, share the script without the token and provide instructions for securely setting it up.
+
+      If you accidentally share your token with someone or include it in version control like Git, immediately revoke your token by expiring it on the [IBM Quantum Platform dashboard](https://quantum.ibm.com/) (click the refresh button in the API token field) to reduce risk.
+    </Admonition>
+
+#### Use a temporary access token
+
+1. Optionally obtain a temporary access token by supplying your API token. This is especially useful if you would like control over tokens, such as token invalidation.
```

Comment:
```suggestion
1. Optionally obtain a temporary access token by supplying your API token. This is especially useful if you would like control over tokens, such as token invalidation. Follow the same precautions as above with securely using the environment variable.
```

(I think we aren't supposed to say "above"?)

--

You also will want to update the code to use `import os` and read in the environment variable like you did above. 

While you're fixing this code, can you please rename `input` to be `papyload`, and have a space between the equals like `payload = {`? `input` is a builtin value in Python and we're overwriting it, which is a bad practice and messes up code highlighting

Reply:
done in 065dd390187b427bc9aa7736439afcf90691622e